make-checkout

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -e
```

Comment:
Please put back the `set -u` -- every serious shell script ought to have that. Why did you drop it? At first sight the only problematic thing is `[ -z "$REPO" ]`, and that can be fixed with [ -z "${REPO:-}" ]

Reply:
yes, that was the exact reason.

I should really learn shell programming as I am doing terrible job :/

Reply:
> learn shell programming

Aren't you doing that *right now*? :grin: